stmhal/boards/STM32F429DISC/mpconfigboard.h

```diff
@@ -29,6 +29,24 @@
 #define MICROPY_HW_I2C1_SCL (pin_A8)
 #define MICROPY_HW_I2C1_SDA (pin_C9)
 
+// SPI busses
+//#define MICROPY_HW_SPI1_NSS     (pin_A4)
+//#define MICROPY_HW_SPI1_SCK     (pin_A5)
+//#define MICROPY_HW_SPI1_MISO    (pin_A6)
+//#define MICROPY_HW_SPI1_MOSI    (pin_A7)
```

Comment:
Definition of SPI2 collides with usage of USE_USB_HS_IN_FS. Therfore I do not add it here.